portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,7 +26,26 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    // Convert to JSON
+    String message1 = "My name is Micayla.";
+    String message2 = "I am from the Bay Area.";
+    String message3 = "I live in New York City.";
```

Comment:
I don't think there's much value in having these strings being local variables, given that they're only used once. I'd recommend just doing

```
micaylaMessages.add("My name is Micayla.");
```

etc.  If we were using these string constants elsewhere, we'd want to make them static class members:

```
private static final String MICAYLA_NAME = "My name is Micayla.";
...
```

but again, no need for that here.

Reply:
I see that you changed this in your commit. It's always a good idea to respond in some way to reviewer comments. In this case, a simple "Done" would be just fine. This helps set my expectations for what I should see when I review your updated PR.